pkg/transforms/coredata.go

```diff
@@ -0,0 +1,24 @@
+package transforms
+
+import (
+	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
+)
+
+type CoreData struct {
+}
+
+// NewCoreData Is provided to interact with CoreData
+func NewCoreData() *CoreData {
+	coredata := new(CoreData)
```

Comment:
QQ: why use of `new` rather than CoreData{} like we have in the others?

Reply:
Because it was copied from the other tests we have, i.e MqttConfig

Reply:
We should be consistent across all the functions and based on this we should be using CoreData{}
https://softwareengineering.stackexchange.com/questions/210399/why-is-there-a-new-in-go